README.md

```diff
@@ -1,193 +1,266 @@
-  Modelling of Biochemical Reaction
-  Networks](https://www.youtube.com/watch?v=s1e72k5XD6s)
-
-Finally, an overview of the package and its features (as of version 13) can also be found in its corresponding research paper, [Catalyst: Fast and flexible modeling of reaction networks](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011530).
+An overview of the package, its features, and comparative benchmarking (as of version 13) can also 
+be found in its corresponding research paper, [Catalyst: Fast and flexible modeling of reaction networks](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011530).
 
 ## Features
 
-- A DSL provides a simple and readable format for manually specifying chemical
-  reactions.
-- Catalyst `ReactionSystem`s provide a symbolic representation of reaction networks,
-  built on [ModelingToolkit.jl](https://docs.sciml.ai/ModelingToolkit/stable/) and
-  [Symbolics.jl](https://docs.sciml.ai/Symbolics/stable/).
-- Non-integer (e.g. `Float64`) stoichiometric coefficients are supported for generating
-  ODE models, and symbolic expressions for stoichiometric coefficients are supported for
-  all system types.
-- The [Catalyst.jl API](http://docs.sciml.ai/Catalyst/stable/api/catalyst_api) provides functionality for extending networks,
-  building networks programmatically, network analysis, and for composing multiple
-  networks together.
-- `ReactionSystem`s generated by the DSL can be converted to a variety of
-  `ModelingToolkit.AbstractSystem`s, including symbolic ODE, SDE and jump process
-  representations.
-- Coupled differential and algebraic constraint equations can be included in
-  Catalyst models, and are incorporated during conversion to ODEs or steady
-  state equations.
-- Conservation laws can be detected and applied to reduce system sizes, and generate
-  non-singular Jacobians, during conversion to ODEs, SDEs, and steady state equations.
+#### Features of Catalyst
+- [The Catalyst DSL](@ref ref) provides a simple and readable format for manually specifying reaction 
+ network models using chemical reaction notation.
+- Catalyst `ReactionSystem`s provides a symbolic representation of reaction networks,
+ built on [ModelingToolkit.jl](https://docs.sciml.ai/ModelingToolkit/stable/) and
+ [Symbolics.jl](https://docs.sciml.ai/Symbolics/stable/).
+- The [Catalyst.jl API](http://docs.sciml.ai/Catalyst/stable/api/catalyst_api) provides functionality 
+ for extending networks, building networks programmatically, and for composing 
+ multiple networks together.
+- Generated models can be simulated using any
+ [DifferentialEquations.jl](https://docs.sciml.ai/DiffEqDocs/stable/)
+ [ODE/SDE/jump solver](@ref ref), and can be used within `EnsembleProblem`s for carrying
+ out [parallelized parameter sweeps and statistical sampling](@ref ref). Plot recipes
+ are available for [visualization of all solutions](@ref ref).
```

Comment:
I tried to keep American English here (so that we are consistent within the page). If you see anything I miswrote, feel free to point it out (or not to worry about it either. For a while I didn't really know what to do here, an in the end I decided to just try and be consistent within each file, mostly so that I know what to do, but if we end up mixing I do not mind either).